example.js

```diff
@@ -119,8 +121,18 @@ shippo.customsdeclaration.create({
 	process.exit(1);
 }).then(function(rates) {
 	console.log("rates : %s", JSON.stringify(rates, null, 4));
-	// Get the first rate in the rates results for demo purposes.
-	rate = rates.results[0];
+	try {
+		const filteredRates = rates.results.filter(rate => rate.provider.toUpperCase().includes('USPS') || rate.provider.toUpperCase().includes('DHL Express'));
```

Comment:
The toUpperCase() won't work for `DHL Express`

Reply:
Corrected